scripts/variantstore/scripts/create_ranges_cohort_extract_data_table.py

```diff
@@ -425,6 +439,9 @@ def make_extract_table(call_set_identifier,
 
     args = parser.parse_args()
 
+    if not args.vet_ranges_extract_table_version:
```

Comment:
Either here or in the parser we might want to validate that we actually got `V1`, `V2`, or nothing at all

Reply:
Yeah, that's a good point. Will do!